rundeckapp/grails-spa/packages/ui-trellis/src/library/utilities/uivi18n.ts

```diff
@@ -1,60 +1,21 @@
-import ar_EG from "uiv/src/locale/lang/ar-EG";
-import bg_BG from "uiv/src/locale/lang/bg-BG";
-import cs_CZ from "uiv/src/locale/lang/cs-CZ";
-import de_DE from "uiv/src/locale/lang/de-DE";
-import en_US from "uiv/src/locale/lang/en-US";
-import es_ES from "uiv/src/locale/lang/es-ES";
-import fi_FI from "uiv/src/locale/lang/fi-FI";
-import fr_FR from "uiv/src/locale/lang/fr-FR";
-import hu_HU from "uiv/src/locale/lang/hu-HU";
-import it_IT from "uiv/src/locale/lang/it-IT";
-import ja_JP from "uiv/src/locale/lang/ja-JP";
-import nb_NO from "uiv/src/locale/lang/nb-NO";
-import nl_NL from "uiv/src/locale/lang/nl-NL";
-import pt_BR from "uiv/src/locale/lang/pt-BR";
-import ro_RO from "uiv/src/locale/lang/ro-RO";
-import ru_RU from "uiv/src/locale/lang/ru-RU";
-import sv_SE from "uiv/src/locale/lang/sv-SE";
-import tr_TR from "uiv/src/locale/lang/tr-TR";
-import zh_CN from "uiv/src/locale/lang/zh-CN";
+// import ar_EG from "uiv/src/locale/lang/ar-EG";
+// import bg_BG from "uiv/src/locale/lang/bg-BG";
+// import cs_CZ from "uiv/src/locale/lang/cs-CZ";
+// import de_DE from "uiv/src/locale/lang/de-DE";
+// import en_US from "uiv/src/locale/lang/en-US";
+// import es_ES from "uiv/src/locale/lang/es-ES";
+// import fi_FI from "uiv/src/locale/lang/fi-FI";
+// import fr_FR from "uiv/src/locale/lang/fr-FR";
+// import hu_HU from "uiv/src/locale/lang/hu-HU";
+// import it_IT from "uiv/src/locale/lang/it-IT";
+// import ja_JP from "uiv/src/locale/lang/ja-JP";
+// import nb_NO from "uiv/src/locale/lang/nb-NO";
+// import nl_NL from "uiv/src/locale/lang/nl-NL";
+// import pt_BR from "uiv/src/locale/lang/pt-BR";
+// import ro_RO from "uiv/src/locale/lang/ro-RO";
+// import ru_RU from "uiv/src/locale/lang/ru-RU";
+// import sv_SE from "uiv/src/locale/lang/sv-SE";
+// import tr_TR from "uiv/src/locale/lang/tr-TR";
+// import zh_CN from "uiv/src/locale/lang/zh-CN";
 
-export default {
-  ar_EG,
-  ar: ar_EG,
-  bg_BG,
-  bg: bg_BG,
-  cs_CZ,
-  cs: cs_CZ,
-  de_DE,
-  de: de_DE,
-  en_US,
-  en: en_US,
-  es_ES,
-  es: es_ES,
-  fi_FI,
-  fi: fi_FI,
-  fr_FR,
-  fr: fr_FR,
-  hu_HU,
-  hu: hu_HU,
-  it_IT,
-  it: it_IT,
-  ja_JP,
-  ja: ja_JP,
-  nb_NO,
-  nb: nb_NO,
-  nl_NL,
-  nl: nl_NL,
-  pt_BR,
-  pt: pt_BR,
-  ro_RO,
-  ro: ro_RO,
-  ru_RU,
-  ru: ru_RU,
-  sv_SE,
-  sv: sv_SE,
-  tr_TR,
-  tr: tr_TR,
-  zh_CN,
-  zh: zh_CN,
-};
+export default {};
```

Comment:
uiv doesn't export those anymore - will possibly just delete the file once the rest is working